cmd/bootkube/start.go

```diff
@@ -23,15 +24,17 @@ var (
 	}
 
 	startOpts struct {
-		assetDir   string
-		etcdServer string
+		assetDir       string
+		etcdServer     string
+		selfHostedEtcd bool
 	}
 )
 
 func init() {
 	cmdRoot.AddCommand(cmdStart)
 	cmdStart.Flags().StringVar(&startOpts.etcdServer, "etcd-server", "http://127.0.0.1:2379", "Single etcd node to use during bootkube bootstrap process.")
 	cmdStart.Flags().StringVar(&startOpts.assetDir, "asset-dir", "", "Path to the cluster asset directory. Expected layout genereted by the `bootkube render` command.")
+	cmdStart.Flags().BoolVar(&startOpts.selfHostedEtcd, "self-hosted-etcd", false, "Self hosted etcd mode. Includes starting the initial etcd member by bootkube.")
```

Comment:
Can we change this to `--experimental-self-hosted-etcd` or at least add "experimental" to the help text.